teuthology/orchestra/remote.py

```diff
@@ -401,6 +401,17 @@ def get_file(self, path, sudo=False, dest_dir='/tmp'):
             self.remove(path)
         return local_path
 
+    def write_file(self, path, data):
```

Comment:
I think it would be more usable have the same style as 'put_file' and 'get_file', i.e. give the name `put_data` or `get_data` correspondingly.

Reply:
Sorry, you are unclear to me here. I can't see `get_data` and `put_data` in [`get_file`](https://github.com/ceph/teuthology/blob/master/teuthology/orchestra/remote.py#L364) and [`put_file`](https://github.com/ceph/teuthology/blob/master/teuthology/orchestra/remote.py#L354).

Reply:
@rishabh-d-dave I meant create new methods `get_data` and `put_data`, which can accept string, byte or file object as source argument instead of local file path as in `write_file` method.

Reply:
> @rishabh-d-dave I meant create new methods `get_data` and `put_data`, which can accept string, byte or file object as source argument instead of local file path as in `write_file` method. I hope this helps.

Ok, this helps but I am only partially clear. Is it necessary to do it on this PR? I only wanted to get `write_file` merged so that I can test [this PR](https://github.com/ceph/ceph/pull/28692/) on teuthology.

Reply:
@rishabh-d-dave The common approach is to test the code before merge, so one should avoid merging before testing the code. My suggesting to give a name get/put to keep API style, if it is easy to do, why not to have it in a PR. Giving better names earlier, can minimize refactoring in future. Having 'write_file and put_file' makes it a confusion for coder which method to use, until discover whole method signature. Having pairs 'put_file and put_data' makes it clear at least for me, that in first case I'm going to use a filepath as a source, and in seconds case it is python data as a source. Hope it makes clear.